regression/goto-instrument/nondet_static_exclude/test.desc

```diff
@@ -0,0 +1,18 @@
+CORE
+main_1.c main.c 
+--nondet-static --nondet-static-exclude test1 --nondet-static-exclude test3 --nondet-static-exclude test4 --nondet-static-exclude main_1.c:value
+^VERIFICATION FAILED$
+^EXIT=10$
+^SIGNAL=0$
+assertion test1 == (?:NULL||0): SUCCESS
```

Comment:
Why such elaborate pattern matching?

Reply:
Because in some test in CI this was NULL and in other it was 0.

What would you expect to be the default value in CI for all compilers?

Reply:
That seems fine, but wouldn't then `(NULL|0)` do instead of `(?:NULL||0)`?

Reply:
Yes